### PR TITLE
AJAX calls for last fm api

### LINKED ISSUE #19: JS event listeners

### LINKED ISSUE #20: JS Ajax calls

### PR DESCRIPTION
**Pull request by drg0drg**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,19 +41,19 @@ <h4 class="white-text">search for all things, music</h4>
       <form action="#" id="query-selection">
         <p>
           <label>
-            <input name="group1" type="radio" class="with-gap" checked />
+            <input name="group1" type="radio" class="with-gap track-btn" checked />
```
Comment (Goldenaries658) [inline index.html]:
```suggestion
            <input id="track-btn" name="group1" type="radio" class="with-gap" checked />
```

File: index.html [hunk 2]
```diff
@@ -41,19 +41,19 @@ <h4 class="white-text">search for all things, music</h4>
...
             <span class="query-type white-text">Track</span>
           </label>
         </p>
         <p>
           <label>
-            <input name="group1" type="radio" class="with-gap" />
+            <input name="group1" type="radio" class="with-gap album-btn" />
```
Comment (Goldenaries658) [inline index.html]:
```suggestion
            <input id="album-btn" name="group1" type="radio" class="with-gap" />
```

File: index.html [hunk 3]
```diff
@@ -41,19 +41,19 @@ <h4 class="white-text">search for all things, music</h4>
...
             <span class="query-type white-text">Album</span>
           </label>
         </p>
         <p>
           <label>
-            <input name="group1" type="radio" class="with-gap" />
+            <input name="group1" type="radio" class="with-gap artist-btn" />
```
Comment (Goldenaries658) [inline index.html]:
```suggestion
            <input id="artist-btn" name="group1" type="radio" class="with-gap" />
```

File: assets/script.js [hunk 4]
```diff
@@ -1,3 +1,83 @@
...
+})
+
+
+
+function callAPI(){
+    if($('.track-btn').is(':checked')) {
+        getTrackInfo(userInput);
+    } else if ($('.album-btn').is(':checked')) {
+        getAlbumInfo(userInput);
+    } else if ($('.artist-btn').is(':checked')) {
```
Comment (Goldenaries658) [inline assets/script.js]:
```suggestion
    if($('#track-btn').is(':checked')) {
        getTrackInfo(userInput);
    } else if ($('#album-btn').is(':checked')) {
        getAlbumInfo(userInput);
    } else if ($('#artist-btn').is(':checked')) {
```

**Review by Goldenaries658 — CHANGES_REQUESTED** [review]
Change classes to IDs for good practice as they will be unique. Please format before pushing.

File: index.html (hunk 1 above)
Comment (drg0drg) [reply index.html]: changed the class to id

**Review by meeday — APPROVED** [review]

**Review by meeday — APPROVED** [review]

**Review by Goldenaries658 — APPROVED** [review]

**Pull request merged**